bundler.d/assets.rb

```diff
@@ -17,4 +17,5 @@
   gem 'gridster-rails', '~> 0.1'
   gem 'jquery_pwstrength_bootstrap', '~> 1.2'
   gem 'jquery-turbolinks', '~> 2.1'
+  gem 'select2-rails'
```

Comment:
Please pin this to an appropriate version range, check how the project versions their releases.